modules/apps/accessibility/accessibility-menu-web/types/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export {accessibilityMenuAtom} from './AccessibilityMenu';
+export declare const CONSTANTS: {
+	ACCESSIBILITY_SETTING_EXPAND_TEXT: string;
+	ACCESSIBILITY_SETTING_INCREASE_TEXT_SPACING: string;
+	ACCESSIBILITY_SETTING_REDUCED_MOTION: string;
+	ACCESSIBILITY_SETTING_SHOW_UNDERLINE: string;
```

Comment:
The constants have been renamed for consistency, so rebasing and renaming here to match is needed. https://github.com/liferay/liferay-portal/blob/master/modules/apps/accessibility/accessibility-menu-web/src/main/java/com/liferay/accessibility/menu/web/internal/constants/AccessibilitySettingConstants.java